src/data.ts

```diff
@@ -196,7 +198,7 @@ export interface SphereGenerator extends GeneratorBase {
   /**
    * Generate sphere GeoJSON data for the full globe.
    */
-  sphere: true | {};
+  sphere: true | Dict<any>;
```

Comment:
```suggestion
  sphere: true | Dict<unknown>;
```

Or should this be `object` so we allow interface types?